src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceModelHttpMessageHandler.NetNative.cs

```diff
@@ -53,8 +41,8 @@ public bool UseDefaultCredentials
 
         public bool UseProxy
         {
-            get { return _innerHandler.UseProxy; }
-            set { _innerHandler.UseProxy = value; }
+            get { return true; }
+            set { /* RTHttpClient can only use the system defined proxy so this is a no-op */ }
```

Comment:
This is the same behavior as we had before as this is copied from HttpClientHandler in netcore50. _innerHandler.UseProxy used to call into the netcore50 implementation of HttpClientHandler.